sirepo/package_data/static/js/sirepo-lattice.js

```diff
@@ -507,8 +507,12 @@ SIREPO.app.service('rpnService', function(appState, requestSender, $rootScope) {
         if (self.isCaseInsensitive) {
             v = v.toLowerCase ? v.toLowerCase() : v;
         }
-        if (appState.models.rpnCache && v in appState.models.rpnCache) {
-            return appState.models.rpnCache[v];
+        // TODO(e-carlin): This is the crux. rpnCache is now under externalLattice
```

Comment:
@moellep the whole rpnService needs to change to support the rpnCache/variables being under externalLattice. Any advice on how best to go about this?

Reply:
I don't think we want any variable values in the external lattice. Part of the fixups which remove the command entries should also go through and evaluate every field and replace any rpn expressions with a float value.